### DISCUSSION TITLE
Debugging double escaping of content

### DISCUSSION THREAD

**Discussion by reinhard-brandstaedter** [Dynatrace/dynatrace-configuration-as-code#1603] [Q&A]
Hi,

is there any general recommended approach how to debug the rendering process of JSON templates with monaco. I'm having a hard time when for example pushing workflows that contain configuration for MS Teams notifications, which in turn contain messages in JSON format, which in turn contain template configs. Double, triple escaping?

Is there a way to output e.g. the monaco-rendered JSON config files to see what the actual payload that would be sent to the API would be? Otherwise it is almost trial-and-error to debug.

**Comment by Laubi** [comment]
Hi @reinhard-brandstaedter , there is currently not a possibility to do that, except compiling the project yourself and debugging the code.
